packages/perseus-editor/src/components/json-editor.tsx

```diff
@@ -1,30 +1,62 @@
 /* eslint-disable @babel/no-invalid-this */
 /* eslint-disable react/no-unsafe */
-import createReactClass from "create-react-class";
 import * as React from "react";
 import _ from "underscore";
 
-const JsonEditor: any = createReactClass({
-    displayName: "JsonEditor",
+type Props = {
+    multiLine: boolean;
+    value: any;
+    onChange: (newJson: any) => void;
+};
 
-    getInitialState: function () {
+type DefaultProps = {
+    value: Props["value"];
+};
+
+type State = {
+    currentValue: string | undefined;
+    valid: boolean | undefined;
+};
+
+class JsonEditor extends React.Component<Props, State> {
+    static displayName: "JsonEditor";
+
+    static defaultProps: DefaultProps = {
+        value: 0,
```

Comment:
This seems a bit odd.  We `JSON.stringify()` `value` in other parts, so the default thing we'd edit is `"0"`?